### ISSUE TITLE
histogram help does not match implementation

### ISSUE THREAD

**Issue by RPCMoritz** [FRosner/spawncamping-dds#165]
"The number of buckets parameter is optional having the default value of 10."

```
def histogram[N: ClassTag](values: RDD[N], numBuckets: Int = 100)(implicit num: Numeric[N]): Unit = {
```

**Comment by FRosner** [comment]
@RPCMoritz can you fix it?

**Comment by RPCMoritz** [comment]
I will fix it in the course of adding #92. Should a default value remain an option at all? Or should we ask for an explicit value, mapping 0 to using Sturge's formula?

**Comment by FRosner** [comment]
I am no fan of implicit defaults like negative numbers or something. I suggest we treat sturges as default because it should give a better result. If the user wants a different binning without sturges, he can specify the value explicitly. So I close this one up and we adjust the documentation of the default value that it uses sturges formula in #92.
